Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml

```diff
@@ -0,0 +1,105 @@
+category: Data Enrichment & Threat Intelligence
+fromversion: 5.5.0
+commonfields:
+  id: BruteForceBlocker Feed
+  version: -1
+configuration:
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
```

Comment:
You have `proxy` and `insecure` twice in the YAML.

Reply:
fixed

Reply:
you should have kept the ones at the bottom. This is not a good place for having them

Reply:
done